src/gondi.ts

```diff
@@ -357,8 +357,13 @@ export class Gondi {
     loanId: bigint;
     nftReceiver?: Address;
   }) {
+    const startTime = loan.startTime as Date | bigint;
```

Comment:
I don't like this. Maybe change the type of the input loan. `loan: LoanV4V4 & {startTime: Date | bigint};` or smth like that